x509_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"net/http/httptest"
 	"strings"
 	"testing"
+	"time"
+
+	"golang.org/x/net/context"
```

Comment:
use context from the stdlib. this is the old package and contains a type alias for the stdlib type.

Reply:
also we can remove the additional module requirement with this no longer imported, so revert go,{mod,sum} back to before.

Reply:
Done.  Not sure why goreturns added that instead of the stdlib version.